src/components/carmel-gearvr-controls.js

```diff
@@ -0,0 +1,200 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var getGamepadsByPrefix = require('../utils/tracked-controls').getGamepadsByPrefix;
+var THREE = require('../lib/three');
+
+var GAMEPAD_ID_PREFIX = 'Gear VR Touchpad';
+
+/**
+ * Vive Controls Component
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('carmel-gearvr-controls', {
+  // since aframe-teleport-controls looks for tracked-controls, add one
+  dependencies: ['tracked-controls'],
```

Comment:
+1, but at a higher level, would this make more sense in the registry vs core? I think it's valuable to include, in either case.

Also wondering if the name (and functionality) could be broader than carmel and gearvr. Assuming this is purely using gamepad API, this could be `trackpad-controls` or `dpad-controls`.